conf/application.conf

```diff
@@ -71,10 +71,10 @@ microservice {
     }
 
     client {
-      id = "customs-declare-imports-frontend"
+      id = "customs-declare-exports-frontend"
```

Comment:
@raghu1978 Confirm that we should change it here or not?

Reply:
I think imports doesn't use the stub so we might use it for our purpose let Chris know and we should be fine.